### DISCUSSION TITLE
Add a checkmark for 'Hysteresis only applies on the way up'

### DISCUSSION THREAD

**Discussion by ghostcreeper** [Rem0o/FanControl.Releases#2422] [Ideas]
I think this would solve my issue: cpu % / fans tend to spike randomly from time to time with various browsing usage. I want this option so that the fans ramp up only if the cpu spikes for a prolonged period of time (i.e. during gameplay) but would instantly go down with load % without requiring a response time. I have a step up:step down ratio of 1:20.

**Comment by ionGL** [comment]
I think a time average custom sensor would help your spikes situation. Put a 20-30 seconds value and it will be smooth.
![timeaverageexample](https://github.com/Rem0o/FanControl.Releases/assets/80151979/6a9c94a7-c869-49f7-a63c-13bd5f2a640d)

**Comment by Rem0o** [comment]
That's really against the common "best practices" for cooling.   You want it to lag when it heats up and instantly go down?  Why not simply use normal symmetric hysteresis ? That will cover the spikes, and only ramp up if you get prolonged heat.

**Reply by joaosump** [reply]
![Screenshot 2025-04-09 161828](https://github.com/user-attachments/assets/f6f3fcc7-d417-4129-b7ae-5645492c29ae)

Armoury Crate has such functionality and I think it's very useful specially when you want your pc to run as quiet as possible. It happens often while playing games or simply browsing the internet that your CPU randomly spikes the temps for 3-4 secs, and if you would have set an immediate fan response you would have this annoying noise of the fans spinning faster for 2 secs and it wouldn't matter at all for temps because when the fan speed has ramped up the CPU temp has already normalized.
